support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.tsx

```diff
@@ -145,19 +143,25 @@ const StripeForm = (props: StripeFormPropTypes) => {
 		}
 	};
 
-	const handleCardSetup = (clientSecret: Option<string>): Promise<string> => {
-		const cardElement = elements.getElement(CardNumberElement);
-		return stripe.handleCardSetup(clientSecret, cardElement).then((result) => {
-			if (result.error) {
-				handleStripeError(result.error);
-				return Promise.reject(result.error);
-			}
+	const handleCardSetup = (clientSecret: Option<string>) => {
+		const cardElement = elements?.getElement(CardNumberElement);
+		if (stripe && clientSecret && cardElement) {
+			return stripe
+				.confirmCardSetup(clientSecret, {
```

Comment:
here `stripe.handleCardSetup` has changed to `stripe.confirmCardSetup` - is that expected?

Reply:
Yes, as I explained in the intro comment for the PR, `handleCardSetup` is deprecated (and does not have any types), and `confirmCardSetup` is the replacement method. There's links to the Stripe docs on both above.

Reply:
Oh haha I don't know how I missed that! Nice one 👍